dstep/translator/IncludeHandler.d

```diff
@@ -84,62 +92,120 @@ class IncludeHandler
         ];
     }
 
+    this (Options options)
+    {
+        import std.format;
+
+        this.options = options;
+
+        if (options.packageName != "")
+        {
+            auto inputFiles = options.inputFiles.filter!(
+                x => x != options.inputFile);
+
+            foreach (file; inputFiles)
+                submodules[file] =
+                    fullModuleName(options.packageName, file);
+        }
+    }
+
     void addInclude (string include)
     {
-        rawIncludes ~= include;
+        import std.path;
+        import std.file;
+        import std.array;
+
+        auto absolute = include.asAbsNormPath;
+
+        if (absolute != options.inputFile)
```

Comment:
Yes, it is set in Application.d:69 . [https://github.com/jacob-carlborg/dstep/pull/68/files#diff-906a98df53e7c5cece17f35a0240be48R69](link)